### PR TITLE
pjs/3

### PR DESCRIPTION
**Pull request by korJiusngPark**
solved step3

### CODE REVIEW

**Review by cjy9306 — APPROVED** [review]
👍

**Review by jukqaz — APPROVED** [review]
💯

File: 3/pjs/10_11720_pjs.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	{
+		cin >> a;
+	
+		if (1 <= a <= 100)
+			break;
+	}
+	for (int i = 0; i < a; i++) 
+	{
+		cin >> b;
+		sum += (b - '0');
```
Comment (AlvinYou) [inline 3/pjs/10_11720_pjs.cpp:20]: WOW, C style sum is very awesome

**Review by AlvinYou — APPROVED** [review]
👍

**Review by kor-haru — APPROVED** [review]
👍

**Pull request merged**